### PR TITLE
New: Rename Import to Library Import

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Subject

#### Todos
- [ ] Tests-N/A
- [X] Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js [hunk 1]
```diff
@@ -78,7 +78,10 @@ class ImportSeriesSelectFolder extends Component {
                       Make sure that your files include the quality in their filenames. eg. <span className={styles.code}>episode.s02e15.bluray.mkv</span>
                     </li>
                     <li className={styles.tip}>
-                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span>
+                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span> Additionally, each movie must be in its own folder within the root/library folder.
+                    </li>
+                    <li className={styles.tip}>
+                      Do not use this Import method to import downloads from your download client. This is ONLY for organized existing libraries.
```
Comment (markus101) [inline frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js:84]: I think “existing organized libraries” is better.

File: frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js [hunk 2]
```diff
@@ -78,7 +78,10 @@ class ImportSeriesSelectFolder extends Component {
                       Make sure that your files include the quality in their filenames. eg. <span className={styles.code}>episode.s02e15.bluray.mkv</span>
                     </li>
                     <li className={styles.tip}>
-                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span>
+                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span> Additionally, each movie must be in its own folder within the root/library folder.
```
Comment (markus101) [inline frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js:81]: Is movie support coming in a separate PR?

**Review by markus101 — CHANGES_REQUESTED** [review]
Series, not movies.

File: frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js (hunk 2 above)
Comment (bakerboy448) [reply frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js:81]: pretty sure there's no plans for that feature at this time xD

File: frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js [hunk 3]
```diff
@@ -78,7 +78,10 @@ class ImportSeriesSelectFolder extends Component {
                       Make sure that your files include the quality in their filenames. eg. <span className={styles.code}>episode.s02e15.bluray.mkv</span>
                     </li>
                     <li className={styles.tip}>
-                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span>
+                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span> Additionally, each series must be in its own folder within the root/library folder.
+                    </li>
+                    <li className={styles.tip}>
+                      Do not use this Import method to import downloads from your download client. This is ONLY for existing organized libraries.
```
Comment (markus101) [inline frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js:84]: `Import` should either be `Library Import` or possible reworded to `Do not use for importing downloads from your download client, this is only for existing organized libraries, not unsorted files.`

`ONLY` shouldn't be capitalized, we have better ways to highlight text if necessary, but I don't think it's actually needed here, people will either read it or they won't.

**Review by markus101 — APPROVED** [review]
A couple more suggestions on wording.

**Review by markus101 — CHANGES_REQUESTED** [review]
Misclick...

**Pull request merged**